docs/development.md

```diff
@@ -281,15 +282,50 @@ I0511 10:15:29.593106 244858 hello_world.cc:57] Response: HELLO AGAIN WORLDS!
 
 ### Signing Oak Wasm modules
 
-Oak Wasm modules can be signed using [Ed25519](https://ed25519.cr.yp.to/)
-scheme.
+As described in [Oak Concepts](/docs/concepts.md#labels) Oak Wasm modules can be
+signed using [Ed25519](https://ed25519.cr.yp.to/) scheme.
 
-In order create a signature `hello_world.sign` for a Wasm module
-`hello_world.wasm` use the following script:
+If you are reviewing an Oak module (for example `private_set_intersection.wasm`)
+and want to create a signature for it, first you need a private/public key pair.
+It can be generated using `oak_sign` utility:
+
+```bash
+./scripts/oak_sign generate \
+  --private-key=examples/keys/ed25519/test.key \
+  --public-key=examples/keys/ed25519/test.pub
+```
+
+It generates two key files (`test.key` and `test.pub`) that are encoded using
+[PEM](https://tools.ietf.org/html/rfc1421) format.
+
+Then a Wasm module can be signed using the following command:
 
 ```bash
 ./scripts/oak_sign sign \
-  --private-key=examples/certs/ed25519/test.key \
-  --input-file=examples/hello_world/bin/hello_world.wasm \
-  --signature=examples/hello_world/bin/hello_world.sign
+  --private-key=examples/keys/ed25519/test.key \
+  --input-file=examples/private_set_intersection/bin/private_set_intersection.wasm \
+  --signature=examples/private_set_intersection/bin/private_set_intersection.sign
```

Comment:
```suggestion
  --signature=examples/private_set_intersection/bin/reviewer_1.sign
```
Just a suggestion. See also my suggestion for `signatures.toml` below :)

Reply:
Changed to `signature.pem` simply because we already have this file (in https://github.com/project-oak/oak/pull/1512)